### PR TITLE
Disable elements when new answers are no longer allowed

### PR DESCRIPTION
**Pull request by jonatanschroeder**
Addresses part of #5540, in particular for pl-dropdown, pl-rich-text-editor, pl-file-editor and pl-file-upload.

### CODE REVIEW

File: elements/pl-file-editor/pl-file-editor.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ window.PLFileEditor = function (uuid, options) {
   this.editor.setTheme('ace/theme/chrome');
   this.editor.getSession().setUseWrapMode(true);
   this.editor.setShowPrintMargin(false);
+  this.editor.setReadOnly(!!options.readOnly);
```
Comment (nwalters512) [inline elements/pl-file-editor/pl-file-editor.js:22]: Nit: this should always be defined, so the `!!` shouldn't be necessary.
```suggestion
  this.editor.setReadOnly(options.readOnly);
```

File: elements/pl-rich-text-editor/pl-rich-text-editor.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def render(element_html, data):
             'editor_uuid': uuid,
             'question': data['panel'] == 'question',
             'submission': data['panel'] == 'submission',
-            'read_only': 'true' if data['panel'] == 'submission' else 'false'
+            'read_only': 'true' if (data['panel'] == 'submission' or not data['editable']) else 'false'
```
Comment (nwalters512) [inline elements/pl-rich-text-editor/pl-rich-text-editor.py:71]: Is checking the panel still necessary with the addition of the other condition?

**Review by nwalters512 — APPROVED** [review]
Have not tested, but the code looks good!

File: elements/pl-rich-text-editor/pl-rich-text-editor.py (hunk 2 above)
Comment (jonatanschroeder) [reply elements/pl-rich-text-editor/pl-rich-text-editor.py:71]: Yes, it is needed. `data['editable']` can still be true in the submission panel. In essence, `data['editable']` indicates that the question still allows more submissions, even if the panel being rendered is not the question panel itself.

**Pull request merged**